packages/core/src/components/EmojiPicker/index.tsx

```diff
@@ -103,36 +103,40 @@ export class EmojiPicker extends React.Component<Props & WithStylesProps> {
     };
 
     const messages = {
-      recentlyUsed: T.phrase('Recently Used', {}, 'Emoji picker commonly used feature type'),
-      frequentlyUsed: T.phrase('Frequently Used', {}, 'Emoji picker commonly used feature type'),
-      smileysPeople: T.phrase('Smileys & People', {}, 'Emoji picker group and or category'),
-      animalsNature: T.phrase('Animals & Nature', {}, 'Emoji picker group and or category'),
-      foodDrink: T.phrase('Food & Drink', {}, 'Emoji picker group and or category'),
-      travelPlaces: T.phrase('Travel & Weather', {}, 'Emoji picker group and or category'),
-      activities: T.phrase('Activities', {}, 'Emoji picker group and or category'),
-      objects: T.phrase('Objects', {}, 'Emoji picker group and or category'),
-      symbols: T.phrase('Symbols', {}, 'Emoji picker group and or category'),
-      flags: T.phrase('Flags', {}, 'Emoji picker group and or category'),
-      searchResults: T.phrase('Search results', {}, 'Emoji picker search results label'),
-      none: T.phrase('All emojis', {}, 'Emoji picker label that displays all emojis'),
-      skinNone: T.phrase('No skin tone', {}, 'Emoji picker skin tone label'),
-      skinLight: T.phrase('Light skin tone', {}, 'Emoji picker skin tone label'),
-      skinMediumLight: T.phrase('Medium-light skin tone', {}, 'Emoji picker skin tone label'),
-      skinMedium: T.phrase('Medium skin tone', {}, 'Emoji picker skin tone label'),
-      skinMediumDark: T.phrase('Medium-dark skin tone', {}, 'Emoji picker skin tone label'),
-      skinDark: T.phrase('Dark skin tone', {}, 'Emoji picker skin tone label'),
-      search: T.phrase('Search emojis', {}, 'Emoji picker search input field label'),
-      searchA11y: T.phrase(
-        'Search for emojis by keyword',
+      recentlyUsed: T.phrase('Recently Used', {}, { key: 'lunar.emoji.recentlyUsed' }),
```

Comment:
can you keep the context here too? `{ key: 'lunar.emoji.recentlyUsed', context: 'Emoji picker commonly used feature type'}`

Reply:
I removed the context from the `EmojiPicker` as I feel like the phrase + key name is enough to infer it has to do with emojis.